lib/vast/Frontend/GenAction.cpp

```diff
@@ -43,6 +53,7 @@ namespace vast::cc {
             case output_type::emit_assembly: return "s";
             case output_type::emit_high_level: return "hl";
             case output_type::emit_cir: return "cir";
+            case output_type::emit_mlir: return "mlir";
```

Comment:
either keep only `emit_mlir` or targets for specific dialects, otherwise it will be mess